src/types.ts

```diff
@@ -106,6 +110,12 @@ export interface TramOneElement extends Element {
 	[TRAM_TAG_NEW_EFFECTS]: Effect[];
 	[TRAM_TAG_CLEANUP_EFFECTS]: CleanupEffect[];
 	[TRAM_TAG_STORE_KEYS]: string[];
+
+	// development properties, these are not guarnteed to be on the element
+	[TRAM_TAG_NAME]?: string;
+	[TRAM_TAG_PROPS]?: Props;
+	[TRAM_TAG_CHILDREN]?: Children;
+	[TRAM_TAG_GLOBAL_STORE_KEYS]?: string[];
```

Comment:
even if the app is running in development mode, some properties (like props or children) could be undefined.